src/monitor/cpu_set_monitor.cpp

```diff
@@ -44,6 +44,8 @@ CpuSetMonitor::CpuSetMonitor() : MainMonitor()
         }
     }
 
+    trace_.add_threads(get_comms_for_running_processes());
```

Comment:
Why is it moved up?

Reply:
Because the order in which we do things changed. The old exit_reader would read the name of processes which died during measuring, and then at the end add a processes which are still running. With PERF_RECORD_COMM, we first read the names of the already running processes and then collect the names of all processes that started while measuring. If we would do it like the old exit_reader dit it then we would miss all the processes which just quietly died during measuring